src/Core/Models/Export/Login.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Bit.Core.Models.View;
+
+namespace Bit.Core.Models.Export
+{
+    public class Login
+    {
+        public Login() { }
+
+        public Login(LoginView obj)
+        {
+            if(obj == null)
```

Comment:
Remove all of these `if(obj == null)` checks in the export models. This is a hack that we have to do in TS because they do not allow for ctor overloads. Our empty ctor will allow us to do what TS will not.